### DISCUSSION TITLE
Ground Items: Highlight/ignore items conditionally

### DISCUSSION THREAD

**Discussion by gamer88273** [runelite/runelite#15218] [Q&A] [answered]
Can the ability to use comparison operators in the ground items plugin be added?

For example:
I want to highlight death runes when the quantity is above 30 and ignore them if the quantity is less than 10.
To do so, add "death rune > 30" to the highlight list and "death rune < 10" to the ignore list.

**Comment by Alexsuperfly** [comment, marked as answer]
this already exists, exactly as you have described.

**Reply by gamer88273** [reply]
Wow I'm dumb I must have done something wrong when I first tried it. Thanks!
